src/openforms/forms/tests/e2e_tests/test_form_designer.py

```diff
@@ -1369,3 +1369,193 @@ def assertState():
             self.assertIsNone(form.product)
 
         await assertState()
+
+
+class SelectReuseableFormDefinitionsTests(E2ETestCase):
+    async def test_all_reuseable_form_definition_options_are_available(self):
```

Comment:
reading this test name I'd expect all form definitions to be visible in the dropdown, but the assertions check the opposite?